pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -151,6 +153,10 @@ async def run(self):
             advisory_type = "RHEA" if is_ga else "RHBA"
             for ad in advisories:
                 if advisories[ad] < 0:
+                    if not batch_id:
```

Comment:
How do we want to do pre-releases? 
Think this logic works for GA releases, but if we are trying to ship an operator-prerelease, then not.